coalib/output/ConsoleInteraction.py

```diff
@@ -484,7 +484,7 @@ def require_setting(setting_name, arr, section):
                          following.
     :param section:      The section the action corresponds to.
     """
-    needed = join_names(arr[1:])
+    needed = join_names(arr[1:]) + 'True is the default value.'
```

Comment:
`needed` is a list of all the bears which need said setting. I don't think the default message should go here. You probable want to add it at the end of the question. `Please enter a value for the setting "use_spaces" (True if spaces are to be used instead of tabs.) needed by SpaceConsistencyBear for section "Default" (True is the default):`